templates/students/update_profile.html

```diff
@@ -114,11 +99,12 @@ <h2>
                         </div>
                     </div>
                 </div>
-
+                <!--extra input tag added here beacause input tag in popover doesn't works as it is hidden but can trigger this to work-->
+                <input name="profile_image" type="file" id="profileImage" style="display:none" onchange="imageValidation()">
                 <hr class="mb-3">
                 <div class="row justify-content-center mb-3">
                     <div class="col-md-6">
-                        <button name="submit" value="update-profile" class="btn btn-primary btn-lg btn-block"
+                        <button name="submit" id="submit" value="update-profile" class="btn btn-primary btn-lg btn-block"
```

Comment:
Remove `name`, `id` and `value` attribute from here. This is the cause of form not getting submitted in JS.